examples/calico-cert.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "giantswarm.io/v1"
+kind: Certificate
+metadata:
+  name: "uo91f-calico"
+  namespace: "default"
+spec:
+  clusterID: "uo91f"
+  clusterComponent: "calico"
+  commonName: "calico.uo91f.g8s.eu-west-1.aws.test.private.giantswarm.io"
```

Comment:
@xh3b4sd the Calico domain is missing from clustertpr. Can we add it and is the example correct?